### PR TITLE
Whitespace formatting changes.

### PR DESCRIPTION
**Pull request by mookerji**
Some of the curriculum text is difficult to edit, as is, so I've added line wrapping in the curricular documents, changed some capitalization, and inserted white space into the `project.clj`.

### CODE REVIEW

**Comment by seancorfield** [comment]
With all the line formatting changes, it's really hard to see what the actual changes are in this PR and the "Additions" commit also seems to be just whitespace formatting.

Can you summarize what's actually changed here?

**Comment by davidchambers** [comment]
Could you remove the formatting commit? (`git reset --hard 2a7e85a && git push origin additions --force` should do the trick.) Once this pull request has been merged you could open a separate pull request for the formatting changes.

**Comment by mookerji** [comment]
Sorry, I intended to follow up with clarification. Adding now.

**Comment by seancorfield** [comment]
I went through and reviewed the changes and as far as I can tell, the only actual changes - beyond formatting - seem to Capitalization of headings. If that's correct, I'll :+1: this. @mookerji can you confirm that really is the extent of the changes?

**Comment by mookerji** [comment]
I'm going to squash and doublecheck. One second.

**Comment by mookerji** [comment]
Yes, I can confirm that those are the only formatting changes. Thanks for reviewing it!

**Comment by mookerji** [comment]
Hey, can we merge this? @seancorfield @davidchambers Thanks!

**Comment by davidchambers** [comment]
I don't have permission to merge pull requests in this repository.
